xep-0404.xml

```diff
@@ -132,29 +144,29 @@ This change means that the client will not be able to determine real JID of the
      </p>
     </section2>
 
 
   <section2 topic="JID Map Node" anchor="concepts-nodes">
-    <p>This specification defines a JID Map node, so that administrators can see real JIDs for JID Hidden channels.</p>
+    <p>This specification defines a JID Map node, so that administrators can see JIDs for JID Hidden channels.</p>
     <table caption="JID Map Node">
 
-      <tr><td>JID Map</td><td>'urn:xmpp:mix:nodes:jidmap'</td><td>For storing a list of bare proxy JIDs from the participants node with a 1:1 mapping to the corresponding real JIDs.</td><td>Automatic</td><td>PubSub</td></tr>
+      <tr><td>JID Map</td><td>'urn:xmpp:mix:nodes:jidmap'</td><td>For storing a list of Stable Participant IDs from the participants node with a 1:1 mapping to the corresponding  JIDs.</td><td>Automatic</td><td>PubSub</td></tr>
 
     </table>
 
 
 
 
 
-      <p>The JID Map node is used to associate a proxy bare JID to its corresponding real bare JID.   It is a PubSub node with the 'node' attribute set to 'urn:xmpp:mix:nodes:jidmap'. The JID Map node MUST have one entry for each entry in the Participants node.  This value is added when a user joins the channel and is removed when the user leaves the channel.
-       Each item is identified by proxy bare JID, mapping to the real bare JID.  This node is used to give administrator access to real JIDs and participant access to real JIDs in jid-visible channels.  This node MUST NOT be modified directly using pubsub.
-       In JID Visible channels, all participants MAY subscribe to this node.  In JID Hidden and JID Maybe Visible channels, only administrators can subscribe.   The JID Map node is a permanent node with one item per participant. Information is stored in a &lt;participant/&gt; element qualified by the 'urn:xmpp:mix:anon:0' namespace.  The real JID is stored in a &lt;jid/&gt; child element of the &lt;participant/&gt; element.  </p>
+      <p>The JID Map node is used to associate a Stable Participant ID to its corresponding  bare JID.   It is a PubSub node with the 'node' attribute set to 'urn:xmpp:mix:nodes:jidmap'. The JID Map node MUST have one entry for each entry in the Participants node.  This value is added when a user joins the channel and is removed when the user leaves the channel.
+       Each item is identified by Stable Participant ID mapping to the  bare JID.  This node is used to give administrator access to  JIDs.  This node MUST NOT be modified directly using pubsub.
```

Comment:
I think more than MUST NOT be modified (which suggests clients MUST NOT, but the server might allow it), it's that the server MUST NOT allow modifications using xep60 operations.